build/build.sh

```diff
@@ -32,6 +32,9 @@ if [ "$FULL_BUILD" == "true" ]; then
   python "$BUILD_DIR/set_build.py"
 fi
 
+pip install git+https://github.com/lbryio/lbryschema.git
```

Comment:
actually, since build.sh is just running `pip install -r requirements.txt` to install lbrynet, the lbryum branch should indeed be specified in requirements.txt